tests/Doctrine/Tests/Common/Cache/CacheTest.php

```diff
@@ -128,9 +128,9 @@ public function testSaveMultiple() : void
 
     public function provideDataToCache() : array
     {
-        $obj = new \stdClass();
-        $obj->foo = 'bar';
-        $obj2 = new \stdClass();
+        $obj       = new \stdClass();
```

Comment:
I wonder, this is part of Doctrine cs? Since when?

Reply:
This might lead to reformating many line just due to adding a new variable.

Reply:
Yes, absolutely, but we value readability over conflicts, I'd say.

We always required to align these. If I didn't ask it to the committers, I did it myself when merging patches.

Reply:
I don't mind the additional diff as long as the code block is actually understandable later on.

Reply:
If that would be more readable, I'd be ok with git conflicts. It lead to read in columns, not in lines, which is counter intuitive.

To be honest, I haven't seen this type of code for a while. I see it in Doctrine first time, that's why I'm surprised.

Reply:
Yeah, I use it with all my libs regardless, because from a logical perspective, we're assigning a bunch of stuff all together